ui/components/app/modals/export-private-key-modal/export-private-key-modal.component.js

```diff
@@ -79,7 +103,17 @@ export default class ExportPrivateKeyModal extends Component {
     return (
       <div
         className="export-private-key-modal__private-key-display"
-        onClick={() => copyToClipboard(plainKey)}
+        onClick={() => {
+          this.context.trackEvent({
+            category: EVENT.CATEGORIES.KEYS,
+            event: EVENT_NAMES.KEY_EXPORT_COPIED,
+            properties: {
+              key_type: EVENT.KEY_TYPES.PKEY,
+              copy_method: 'clipboard',
+            },
+          });
+          copyToClipboard(plainKey);
```

Comment:
Just a little thing, but I would move that tracking after the `copyToClipboard(plainKey);` call, because I believe we want to track it only if the `copyToClipboard(plainKey);` call is successful. The same in other places.